### PR TITLE
umbraco simple architecture

### PR DESCRIPTION
**Pull request by SunBuild**
Adding a new template for solution campaign . Umbraco simple web app architecture with storage and sql server

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@SunBuild__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (sumuth). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by singhkay** [comment]
@SunBuild Is this a "simple" counterpart to umbraco-cms-webapp-redis-cache? The name of these should probably be similar so it shows up together with that like some of the other templates that are "simple" & "cluster" versions e.g. opensis, moodle. Also check the CI error

File: simple-umbraco-webapp/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Build a simple Umbraco Web App
```
Comment (singhkay) [inline simple-umbraco-webapp/README.md]: @SunBuild The other template is called "umbraco-cms-webapp-redis-cache" and this one "simple-umbraco-webapp" so they won't show next to each other in the root list of the repo. I suggest calling both something like 

"umbraco-webapp-cluster" and "umbraco-webapp-simple" so they appear together in the list. See example templates following this approach below
image

File: simple-umbraco-webapp/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,456 @@
...
+                "Basic",
+                "Standard",
+                "Premium"
+            ],
+            "defaultValue": "Standard",
+            "metadata": {
+                "description": "SQL DB appServiceTier : Basic,Standard,Premium"
+            }
+        },
+        "NonAdminDatabaseUserName": {
```
Comment (singhkay) [inline simple-umbraco-webapp/azuredeploy.json]: camelCase parameters

File: simple-umbraco-webapp/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,456 @@
...
+                    ],
+                    "properties": {
+                        "endIpAddress": "0.0.0.0",
+                        "startIpAddress": "0.0.0.0"
+                    }
+                }
+            ]
+        },
+        {
+            "apiVersion": "2015-05-01-preview",
```
Comment (singhkay) [inline simple-umbraco-webapp/azuredeploy.json]: Use 2016-01-01 version for storage. There is a breaking change you need to be aware of. See API definition https://msdn.microsoft.com/en-us/library/azure/mt163564.aspx

**Comment by singhkay** [comment]
lgtm, merged

**Pull request merged**